test/failed-directories.test.js

```diff
@@ -330,3 +330,12 @@ test.serial('with rcExtensions, find invalid JS in .foorc.js', (assert) => {
     assert.is(error.name, 'SyntaxError', 'threw correct error type');
   });
 });
+
+test.serial('Configuration file not exist', (assert) => {
```

Comment:
I am sure, It fail  on Windows 10 without the above changes.The specific phenomenon is that the console, `ava` long time no response.